test/sequential/test-net-connect-emfile.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+console.log('Test to check the error message format during EMFILE error');
+
+if (process.platform === 'win32') {
+  console.log('Skipping test, no RLIMIT_NOFILE on Windows.');
+  return;
+}
+
+var totalErrorCount = 0;
+
+const reactToError = function() {
+  totalErrorCount += 1;
+  if (totalErrorCount === 2) {
+    console.log('Test OK');
+    process.exit(0);
```

Comment:
Try to avoid calling process.exit() in tests, it has a habit of hiding errors.

Here, you can probably use `const reactToError = common.mustCall(function() { ... }, 2);`.  Or if you want to keep it this way, please write it as `function reactToError() { ... }`.